### PR TITLE
add fossildb

### PR DESCRIPTION
**Pull request by jstriebel**
@fm3: Do we need anything else? Probably update the braingames-libs version?

### CODE REVIEW

**Comment by fm3** [comment]
Yes, exactly. That braingames-libs version is not yet released, though. For `publish-local` it would currently be `11.3.7-useNetworkVKVStore-SNAPSHOT`

File: deployment/config/environment [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+USER_UID=1000
+USER_GID=1000
+USER_GROUP_1=nogroup
+USER_GROUP_2=nogroup
+USER_GROUP_3=nogroup
+
+BINARY_DATA_PATH="./binaryData"
+
+NEW_RELIC_LICENSE_KEY=70d0
```
Comment (normanrz) [inline deployment/config/environment]: We don't have newrelic integration in public datastores.

**Comment by jstriebel** [comment]
Let's wait until this is used in production.
TODOs:
* upgrade braingames-libs version to fossildb-enabled version
* test the deployment in CircleCI?

**Comment by jstriebel** [comment]
@fm3: Could we show a big warning, if fossildb can not be connected? Or even exit, but maybe it should be run without tracing support

**Comment by jstriebel** [comment]
ready for review @normanrz

**Pull request merged**